Gemfile

```diff
@@ -50,6 +50,7 @@ group :development, :test do
   gem 'capistrano-rails', '~> 1.4', require: false
   gem 'capistrano-rvm'
   gem 'coveralls', require: false
+  gem 'debug', platforms: %i(mri mingw x64_mingw)
```

Comment:
You can remove the pry gems since the debug gem does a good job at replacing them